site/docs/v1/tech/core-concepts/users.adoc

```diff
@@ -731,6 +731,27 @@ image::core-concepts/user-search-json-query.png[User Search with JSON Query.,wid
 
 To learn more about the Elasticsearch search engine in general, view the link:/docs/v1/tech/guides/user-search-with-elasticsearch[Elasticsearch search guide].
 
+[NOTE.since]
+====
+Prior to version 1.48.0 if you attempted to page past 10000 results in the Admin UI you would encounter an error.
+
+Starting in version 1.48.0 you are able to page past the 10000 results limit one page at a time.
+====
+
+When there are more than 10000 results when using the Elasticsearch engine you have the option to navigate past the limit, however you may only do so one page at a time.
+
+image::core-concepts/user-search-extended-a.png[User Search with over 10000 results first page,width=1200,role=bottom-cropped]
```

Comment:
I'm not sure how a screen reader would interpret the thousands separator in a number. A quick search didn't give me a good answer. Maybe it's okay to leave them out of the alt text.
```suggestion
image::core-concepts/user-search-extended-a.png[User Search with over 10,000 results first page,width=1200,role=bottom-cropped]
```

Reply:
asciidoc doesn't like them